### PR TITLE
Tidied pod for Tutorial.

### PR DESCRIPTION
**Pull request by manwar**
Hi Team,

Please review the PR.

Many Thanks.
Best Regards,
Mohammad S Anwar

### CODE REVIEW

File: lib/Dancer/Tutorial.pod [hunk 1]
```diff
@@ -19,7 +19,7 @@ that type of traffic to that specific URL.
 
 This example shows a single HTTP verb "GET" followed by the root URL "/" and an anonymous subroutine which returns
 the string C<"Hello World!">  If you were to run this example, it would display "Hello World!" when you point your
-browser at L<http://localhost:3000>.
+browser at C<http://localhost:3000>.
```
Comment (cromedome) [inline lib/Dancer/Tutorial.pod:22]: I like the other changes, but I am not sure why you changed this one. It's the only thing that is keeping me from giving this a :+1:. Why did you change this link in particular? Thanks!

Comment (manwar) [reply lib/Dancer/Tutorial.pod:22]: To help user not to click the link by mistake ;-)

Comment (manwar) [reply lib/Dancer/Tutorial.pod:22]: Because when they do and they will get error page.

Comment (cromedome) [reply lib/Dancer/Tutorial.pod:22]: Thanks for clarifying. :+1: from me!

**Comment by bigpresh** [comment]
Thanks!
